packages/inngest/src/helpers/env.ts

```diff
@@ -388,3 +396,37 @@ export const getResponse = (): typeof Response => {
   // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-var-requires
   return require("cross-fetch").Response;
 };
+
+/**
+ * Given an unknown value, try to parse it as a `boolean`. Useful for parsing
+ * environment variables that could be a selection of different values such as
+ * `"true"`, `"y"`, or `"1"`.
+ *
+ * If the value could not be confidently parsed as a `boolean` or was seen to be
+ * `undefined`, this function returns `undefined`.
+ */
+export const parseAsBoolean = (value: unknown): boolean | undefined => {
+  if (typeof value === "boolean") {
+    return value;
+  }
+
+  if (typeof value === "number") {
+    return Boolean(value);
+  }
+
+  if (typeof value === "string") {
+    const trimmed = value.trim().toLowerCase();
+
+    if (trimmed === "undefined") {
+      return undefined;
+    }
+
+    if (["true", "1", "y", "yes"].includes(trimmed)) {
```

Comment:
Are you ok with dropping support for `"y"` and `"yes"`? I've never seen anyone set a boolean env var to those -- I've only seen `"true"` and `"1"`